R/input-utils.R

```diff
@@ -1,7 +1,7 @@
 shinyInputLabel <- function(inputId, label = NULL) {
   tags$label(
     label,
-    class = "control-label",
+    class = "control-label col-form-label",
```

Comment:
Maybe we want to shim this instead??

Reply:
I actually think this might be completely wrong... the bs4 docs only use this for horizontal form labels.